requirements_test.txt

```diff
@@ -5,7 +5,7 @@ flake8==4.0.1
 ipython==7.34.0
 isort==5.10.1
 mypy==0.961
-numpy==1.21.6
+numpy==1.23.1
```

Comment:
These all need to be `<` just as in https://github.com/pyvista/pyvista/blob/main/requirements_test.txt.